Applications/WebPortal/WebApp/library/source/pages/account_directory_page/create_account_page.tsx

```diff
@@ -198,6 +287,162 @@ export class CreateAccountPage extends React.Component<Properties, State> {
     this.setState({roles: this.state.roles});
   }
 
+  private onFirstNameChange(newValue: string) {
+    this.state.identity.firstName = newValue;
+    this.setState({ identity: this.state.identity });
+    this.shouldButtonBeEnabled();
+  }
+
+  private onLastNameChange(newValue: string) {
+    this.state.identity.lastName = newValue;
+    this.setState({ identity: this.state.identity });
+    this.shouldButtonBeEnabled();
+  }
+
+  private onUsernameChange(newValue: string) {
+    this.setState({ username: newValue });
+    this.shouldButtonBeEnabled();
+  }
+
+  private onEmailChange(newValue: string) {
+    this.state.identity.emailAddress = newValue;
+    this.setState({ identity: this.state.identity });
+    this.shouldButtonBeEnabled();
+  }
+
+  private onAddressChange(newValue: string) {
+    this.state.identity.addressLineOne = newValue;
+    this.setState({ identity: this.state.identity });
+    this.shouldButtonBeEnabled();
+  }
+
+  private onCityChange(newValue: string) {
+    this.state.identity.city = newValue;
+    this.setState({ identity: this.state.identity });
+    this.shouldButtonBeEnabled();
+  }
+
+  private onProvinceChange(newValue: string) {
+    this.state.identity.province = newValue;
+    this.setState({identity: this.state.identity});
+    this.shouldButtonBeEnabled();
+  }
+
+  private onCountryChange(newValue: Nexus.CountryCode) {
+    this.state.identity.country = newValue;
+    this.setState({identity: this.state.identity});
+    this.shouldButtonBeEnabled();
+  }
+
+  private shouldButtonBeEnabled() {
```

Comment:
Rename this method, should in a name suggests it's a question but this function doesn't return anything.